### ISSUE TITLE
Update component documentation

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6482]
We need to ensure that each component's SCSS file contains the correct HTML and accessibility notes, where necessary.  This issue should be completed at the end of the accessibility milestone.

Known affected components:

- [x] Accordion
- [x] Callout block
- [x] Dropdown
- [x] Footer
- [x] Form errors
- [x] Form group
- [x] Table
- [x] Tooltip
